models/product/src/transformers.ts

```diff
@@ -0,0 +1,62 @@
+import { Reference, TReferenceGraphql } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProduct, TProductGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TProduct, TProduct>('default', {
+    buildFields: [
+      'productType',
+      'masterData',
+      'taxCategory',
+      'state',
+      'createdBy',
+      'lastModifiedBy',
+    ],
+  }),
+  rest: Transformer<TProduct, TProduct>('rest', {
+    removeFields: ['skus'],
+    buildFields: [
+      'productType',
+      'masterData',
+      'taxCategory',
+      'state',
+      'createdBy',
+      'lastModifiedBy',
+    ],
+  }),
+  graphql: Transformer<TProduct, TProductGraphql>('graphql', {
+    buildFields: [
+      'productType',
```

Comment:
nit: `productType` here will still be a reference, even though it's documented as `ProductTypeDefinition`. The care level here is low as we're not using it, but it might confuse future consumers. Perhaps we can set `productType` to `null` via `replaceFields()` and add a note that the type doesn't exist (yet)?

I see this pattern reflected in the Orders PR I made [in this types file](https://github.com/commercetools/test-data/blob/35c352722c3f818645f075c49ddbe00c6503b650/models/order/src/types.ts), but admittedly I might have adhered to the documentation too strictly when the payoff is kind of minimal 🤷‍♂️

Reply:
Wondering if for time-boxing sake I should go back to this later. What do you think?

Reply:
Yeah makes sense! Perhaps add a small note?

Reply:
Strong - I'll surely forget if I don't. 🙏

Reply:
I'm not sure I understand the issue here. In GraphQL, `productType` isn't a reference.

Reply:
I removed the comment but I'll leave myself a note that this should possibly be revisited. This is where the graphql docs break down for me, I was making the assumption that for `productType` there was really only the `draft` and then once the entity was created it was called `ProductTypeDefinition` for whatever reason. Is that somewhat accurate?

Reply:
This is no longer an issue now that the generator defines `productType` as not a reference, I missed that commit 👍